.changeset/tasty-eagles-draw.md

```diff
@@ -0,0 +1,30 @@
+---
+"@spectrum-css/link": major
+---
+
+# Link S2 migration
+
+The link component is now updated with S2 specs, colors, and typography.There is a new inline variant which uses the `.spectrum-Link--inline` class and the default variant is the standalone variant.
```

Comment:
```suggestion
The link component is updated with S2 specifications, colors, and typography. There is a new inline variant which uses the `.spectrum-Link--inline` modifier class and the default is the standalone variant.
```

A couple minor wording updates, can you say more about the "standalone" variant. Will most readers of the changelog know what that means or is there a way to describe it that might be more clear? Or can we link to the design documentation?